### PR TITLE
Google Cloud platform

### PR DESCRIPTION
**Pull request by jerkyrs**
Documentation how to deploy full node on Google Cloud Platform  (Compute)

### CODE REVIEW

File: docs/4-running-iost-node/Google-Compute-Platform.md [hunk 1]
```diff
@@ -0,0 +1,279 @@
...
+```
+sudo su - 
+docker ps
+tail -f /data/iserver/logs/iost.log
+```
+You should see blocks syncing
+![verify_server](assets/4-running-iost-node/GoogleCloudPlatform/verify_server.png)
+
+
+You may need to restart docker container if it was started before you applied the firewall rules, use the CONTAINER ID shown from `docker ps`
```
Comment (QwertyJack) [inline docs/4-running-iost-node/Google-Compute-Platform.md]: I suppose this step is not necessary,

Comment (QwertyJack) [inline docs/4-running-iost-node/Google-Compute-Platform.md]: I suppose this step is not necessary.

File: docs/4-running-iost-node/Google-Compute-Platform.md [hunk 2]
```diff
@@ -0,0 +1,279 @@
...
+```
+docker restart 1234abcd
```
Comment (QwertyJack) [inline docs/4-running-iost-node/Google-Compute-Platform.md]: I'd like to use `docker-compose` instead to restart, that is `docker-compose restart iserver`.

File: docs/4-running-iost-node/Google-Compute-Platform.md [hunk 3]
```diff
@@ -0,0 +1,279 @@
...
+```
+
+You may also need to change your account number in /data/iserver/iserver.yml
+
+To check your node is up query a seednode and look for your public ip
```
Comment (QwertyJack) [inline docs/4-running-iost-node/Google-Compute-Platform.md]: For some reason this way might not work as expected.
The most direct way to check the node is to check whether the height increases in the log file or using `iwallet state`.

File: docs/4-running-iost-node/Google-Compute-Platform.md (hunk 1 above)
Comment (jerkyrs) [reply docs/4-running-iost-node/Google-Compute-Platform.md]: ok updated it to reference docker-compose

File: docs/4-running-iost-node/Google-Compute-Platform.md (hunk 2 above)
Comment (jerkyrs) [reply docs/4-running-iost-node/Google-Compute-Platform.md]: updated to use the name defined in docker-compose.yml

File: docs/4-running-iost-node/Google-Compute-Platform.md (hunk 3 above)
Comment (jerkyrs) [reply docs/4-running-iost-node/Google-Compute-Platform.md]: I have added more information here, i noticed if you query your own node it does not list it as a Peer

**Comment by QwertyJack** [comment]
Well done ! Thanks for your help.

**Pull request merged**